logger/logger.go

```diff
@@ -99,16 +107,27 @@ func InitLogger(cfg *appconf.SuperKeyWorkerConfig) *logrus.Logger {
 		logLevel = logrus.FatalLevel
 	}
 
-	formatter := NewCloudwatchFormatter()
+	formatter := NewCustomLoggerFormatter()
+
+    logOutput := os.Stdout
+    forwardLogsToStderr := ForwardLogsToStderr(cfg.LogHandler)
+
+    if forwardLogsToStderr {
+        logOutput = os.Stderr
+    }
 
 	Log = &logrus.Logger{
-		Out:          os.Stdout,
+		Out:          logOutput,
 		Level:        logLevel,
 		Formatter:    formatter,
 		Hooks:        make(logrus.LevelHooks),
 		ReportCaller: true,
 	}
 
+    if forwardLogsToStderr {
+        return Log
+    }
```

Comment:
Rather than returning early, what if we added another case to the `if` statement on L134. Since I assume if we're forwarding everything to stderr we most likely do not want to be uploading to cloudwatch? 

e.g.: `if key != "" && secret != "" && !forwardLogsToStderr(cfg.LogHandler) {`
...or we could just remove the assignment, I put a suggestion there.